modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: CONCEPT
+:navtitle: Configuring workspace target {orch-namespace}
+:keywords: administration guide, configuring, namespace
+:page-aliases: installation-guide:configuring-namespace-strategies, configuring-namespace-strategies
+
+[id="configuring-workspace-target-namespace_{context}"]
+= Configuring workspace target {orch-namespace}
+
+For each user, {prod-short} isolates the workspaces in a {orch-namespace}.
+You can configure the {orch-namespace} name template.
+
+{prod-short} server checks usernames and IDs for compatibility with {orch-name} objects naming convention before creating a {orch-namespace} from a template:
+
+*  Incompatible usernames or IDs are reduced to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. 
+* The addition of a random 6-symbol suffix prevents IDs from collisions. The result is stored in preferences for reuse.
```

Comment:
id are openshift UIDs AFAIK, I do not think we need any suffix here

Reply:
I observed that Minikube adds the suffix. Therore, valid on Kubernetes too.